ui/admin/app/utils/route-info.js

```diff
@@ -0,0 +1,13 @@
+export function paramValueFinder(resourceName, parentInfo) {
+  if (!parentInfo || parentInfo.localName === resourceName) {
+    return [];
+  }
+  if (parentInfo.params) {
+    let otherParam = Object.values(parentInfo.params);
+    return [
+      ...paramValueFinder(resourceName, parentInfo.parent),
+      ...otherParam,
+    ];
+  }
+  return [paramValueFinder(resourceName, parentInfo.parent)];
```

Comment:
Doesn't this need to be spread as well or just returned as is? Wouldn't we have nested arrays otherwise?

I wonder if it can even be hit since I think `parentInfo.params` returns an empty object which works in the previous return. I think we'd need to check for empty objects otherwise this seems a little misleading.

```suggestion
  return paramValueFinder(resourceName, parentInfo.parent);
```

Reply:
Yup, you are correct, good catch! thank you!